### PR TITLE
fix ci

### PR DESCRIPTION
**Pull request by RadioPotin**

### CODE REVIEW

File: src/frontend/searchAdvanced.ml [hunk 1]
```diff
@@ -785,12 +785,10 @@ let search_page () =
                     )
                     ()
             )
-          ()
-      )
```
Comment (zapashcanon) [inline src/frontend/searchAdvanced.ml:789]: ?

File: .github/workflows/doc-deploy.yml [hunk 2]
```diff
@@ -29,6 +29,7 @@ jobs:
           opam install -y dune odoc json-data-encoding json-data-encoding ezjsonm uuidm geoip js_of_ocaml-lwt calendar tls lwt_log  cohttp-lwt-jsoo ppxlib
       - run: opam pin add . -y --no-action
       - run: opam pin https://github.com/OCamlPro/digodoc-search-api.git
+      - run: opam pin https://github.com/ocaml/omd.git
```
Comment (zapashcanon) [inline .github/workflows/doc-deploy.yml:32]: I don't understand why you're pinning this ?

File: src/frontend/searchAdvanced.ml (hunk 1 above)
Comment (RadioPotin) [reply src/frontend/searchAdvanced.ml:789]: Was added by mistake in commit fb2537613eb96685a6f4061e09662b17dd8d7164

I wont compile because of it

**Comment by RadioPotin** [comment]
I believe that fixing the CI for windows would require to publish a new version of [ez_api](https://github.com/OCamlPro/ez_api)
The package that it uses, `geoip`, cannot compile on windows and there seems to already be a fix on that repository for that matter.

See: [this dune file](https://github.com/OCamlPro/ez_api/blob/c283285479f7a5315d93936255ad8f859878ac62/src/server/dune#L6-L8)

**Pull request closed without merging**